doc/data/messages/d/differing-param-doc/bad.py

```diff
@@ -0,0 +1,10 @@
+def foo(x, y):
```

Comment:
Could you use real world example without foo / bar, please ? It's more lively and understandable for beginner with actual examples.

Reply:
Changed to a real word example